lib/rules/execute-rule-includes.js

```diff
@@ -0,0 +1,18 @@
+const includes = require('lodash.includes');
+const prepForComparison = require('./prep-for-comparison');
+
+function executeRuleIncludes(input, rule, values) {
+  const firstMatch = values.find((value) => {
+    let property = value[rule.property];
+    if (property === undefined) {
+      return;
+    }
+    property = prepForComparison(property, rule);
```

Comment:
what if instead of calling `prepForComparison` in each individual rule, we wrap the operations like `includes` or `===` and have those wrapper functions call `prepForComparison`? Will also consolidate any changes we want in the future to these operations or changes to how `prepping` works

Reply:
good suggestion, i may move that to another PR though